core/croquet/src/main/java/org/lgna/croquet/views/DropDown.java

```diff
@@ -179,11 +185,14 @@ public void paint(Graphics g) {
       if (isActive || DropDown.this.isInactiveFeedbackDesired()) {
         if (isActive) {
           if (buttonModel.isPressed()) {
+            // active
```

Comment:
Comments are a good addition. This logic could be simpler than how it is written.

Instead of nested ifs
Pressed -> gray127
Hover or inactive desired -> gray220
else background

Not asking you to change it, but I spent the thought to mentally untangle it, so I am jotting it down.